airflow/www/static/js/api/useDatasetEvents.ts

```diff
@@ -30,7 +30,7 @@ export default function useDatasetEvents({
   const query = useQuery(
     ['datasets-events', datasetId, sourceDagId, sourceRunId, sourceTaskId, sourceMapIndex, limit, offset, orderBy],
     () => {
-      const datasetsUrl = getMetaValue('dataset_events_api') || '/api/v1/datasets/events';
+      const datasetsUrl = getMetaValue('dataset_events_api');
```

Comment:
Why remove this?

Reply:
Because we get the urls from the `html` file. If these are not working, there is no reason for this single endpoint to keep working while all others (that don't have this 'else' in there) fail. Also, now we need to maintain the API route twice instead of once, so it's neater to remove it IMO.

Reply:
Yeah, `getMetaValue()` was updated to handle `undefined` better. This is fine.

Reply:
Turns out that the grid.html was missing this value.

And due to how `getMetaValue` currently behaves (it just returns an empty string!) this just showed up as "no dataset events" instead.

Temporary fix https://github.com/apache/airflow/pull/27927 but we should go and make getMetaValue error if the requested tag isn't found (but lets do that _after_ 2.5.)

@Jorricks For future: please try to keep entirely unrelated changes like this to a separate PR (this change has 0 to do with notes.)

Reply:
Duly noted. Apologies, next time I'll make sure to move unrelated (thought-to-be) improvements in separate PRs. 
Thanks for getting back to me about this and I hope you didn't spend to much time on this issue.